build/verify-archive.sh

```diff
@@ -10,8 +10,9 @@ set -euo pipefail
 apt-get update
 apt-get install -y cmake xz-utils
 
-tar xzf cockroach.src.tgz
-(cd cockroach-* && make install)
+workdir="$(mktemp -d)"
```

Comment:
This whole file is run within Docker, so theoretically the files will be dropped on container exit, right?

On the other hand, an explicit `rm -rf $workdir` in an exit trap might be nice for readability.